src/views/blocks/tickets/submit-button-modal.php

```diff
@@ -21,20 +24,37 @@
 $button_text = _x( 'Get Tickets', 'Get selected tickets.', 'event-tickets' );
 
 /**
- * Filter Modal Content
+ * Allow filtering of the button classes for the tickets block.
+ *
+ * @since TBD
+ *
+ * @param array $button_name The button classes.
+ */
+$button_classes = apply_filters(
+	'tribe_tickets_ticket_block_submit_classes',
+	[
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: af00d607c447e6cf7737bf3b4e74ba3f</sub>